### ISSUE TITLE
[Bug?] Very strange blank manager cause, moving top menu items with handlers

### ISSUE THREAD

**Issue by exside** [modxcms/revolution#13094]
### Summary

First question is: Am I the only one getting this when doing as explained later on?

The manager goes blank (Internal Server Errors behind, so clearly visible in the inspector) entirely after moving around two menu items that have JavaScript handlers attached, e.g. the one for "flush own permissions" and "reset all sessions (log out all users)", everything is fine a s long as they are in the normal/original parent menu item, but when you move them, let's say into the "User" top nav container (where "Logout" and "Profile" etc. are), an internal server error (500) will happen and the logs say as follows:

```
PHP Fatal error:  Uncaught  --> Smarty Compiler: Syntax error in template "8d6bd625d30e7f6357af45b17d7d9fe19d9a3258"  on line 17 "'success': {fn:function() { location.href = './'; },scope:this}"  - Unexpected ":", expected one of: "}"
```

to see where it actually came from you have to edit this error output of smarty in _core/model/smarty/sysplugins/smarty_internal_templatecompilerbase.php_ at around line 1017, I just added:

```
$error_text .= ', template filepath: ' . "\n" . print_r($this->template->source, 1);
```

to get a hint of what template we are talking about here...because the "path" provided is a hash in this case (for whatever reason)
### Step to reproduce

See above, just move one of the menu items to the user menu and be aware that you have to edit _manager/controllers/default/header.php_ and comment out the section _after_ you have done the menu moving:

``` php
            if (!empty($menu['handler'])) {
                $attributes .= ' onclick="'.str_replace('"','\'',$menu['handler']).'"';
            }
```

because otherwise, you don't have any access to the manager and cannot undo the menu item move...yeah, you could go into the db, but that seemed easier...

The content of both problematic handlers looks fine to me (but could be that some part of the formatting messes withs smarty? but if so, why not when they are in the original top menu?):

``` javascript
MODx.msg.confirm({
    title: _('flush_sessions')
    ,text: _('flush_sessions_confirm')
    ,url: MODx.config.connector_url
    ,params: {
        action: 'security/flush'
    }
    ,listeners: {
        'success': {fn:function() { location.href = './'; },scope:this}
    }
});


MODx.msg.confirm({
    title: _('flush_access')
    ,text: _('flush_access_confirm')
    ,url: MODx.config.connector_url
    ,params: {
        action: 'security/access/flush'
    }
    ,listeners: {
        'success': {fn:function() { location.href = './'; },scope:this}
    }
});
```
### Observed behavior

Manager completely inaccessible/blank/useless
### Expected behavior

Observed behavior not happening =)?
### Environment

MODX 2.5.1-pl advanced install

**Comment by rtripault** [comment]
Just wondering if the proposal in https://github.com/modxcms/revolution/pull/11862 could be helpful ?

**Comment by exside** [comment]
Haha, yeah, I tried, inside the handler code field (maybe that's the wrong place?), but it didn't have an effect.

**Comment by Oetzie** [comment]
I'm going to try and fix this today. #modxbughunt

**Comment by Mark-H** [comment]
Fixed in #13328 for 2.6.0
